lib/libvfio-user.c

```diff
@@ -1729,6 +1729,12 @@ vfu_realize_ctx(vfu_ctx_t *vfu_ctx)
         if (!(vfu_ctx->reg_info[i].flags & VFU_REGION_FLAG_MEM)) {
             vfu_ctx->pci.config_space->hdr.bars[i].io.region_type |= 0x1;
         }
+        if ((vfu_ctx->reg_info[i].flags & VFU_REGION_FLAG_64_BITS)) {
```

Comment:
@jlevon in vfu_setup_region() we don't check whether the size is > 2^32 for 32-bit BARs, I think we should fail if `VFU_REGION_FLAG_64_BITS` isn't set.

Reply:
We can't support bars of that size anyway I think due to the region size uint32_t in private.h